examples/onnx/pytorch/translation/README.md

```diff
@@ -0,0 +1,41 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Usage
+
+This folder contains an example of exporting Bart Model + Beam Search to ONNX file.
```

Comment:
```suggestion
This folder contains an example of exporting BART + Beam Search generation to ONNX.
```
Is the model that is being exported `BartForConditionalGeneration`? Or just `BartModel`?

Reply:
Thanks for the reminder. It's BartForConditionalGeneration.